global.json

```diff
@@ -0,0 +1,9 @@
+{
+  "tools": {
+    "dotnet": "6.0.100"
+  },
+  "msbuild-sdks": {
+    "Microsoft.DotNet.Arcade.Sdk": "7.0.0-beta.22124.1",
+    "Microsoft.DotNet.Helix.Sdk": "7.0.0-beta.22124.1"
```

Comment:
I don't see other references to Helix and, so, you shouldn't need the Helix SDK in this repo.